CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - active/Cross Site WebSocket Hijacking.js > an active scan for Cross-Site WebSocket Hijacking vulnerability
 - targeted/cve-2021-22214.js > A targeted script to check for Unauthorised SSRF on GitLab - CVE 2021-22214
 - httpsender/full-session-n-csrf-nashorn.js > full session and csrf token management.
+- httpfuzzerprocessor/unexpected_responses.js > compare response codes to regex whitelist/blacklist and generate alerts
```

Comment:
```suggestion
- httpfuzzerprocessor/unexpected_responses.js > compare response codes to (a pass/fail) regex and generate alerts
```

Reply:
I made this edit very slightly differently "...a (pass/fail) regex..."